### PR TITLE
Film: truncation test

### PR DESCRIPTION
**Pull request by stephanfowler**
- apply to non-guardian referred Film articles
- retain expanded page's ID in sessionStorage, and don't truncate on return visit
- fix so that clickable elements below the button don't fire
- change button text to "Continue reading...'

### CODE REVIEW

File: static/src/stylesheets/module/content/_truncation.scss [hunk 1]
```diff
@@ -11,19 +11,9 @@
...
-.content__read-more-button {
-    display: none;
-
-    .content__article-body--truncated & {
-        display: block;
-        position: absolute;
-        bottom: 0;
+        height: 40px;
+        background-color: #ffffff;
+        box-shadow: 0 0 120px 60px #ffffff;
```
Comment (sndrs) [inline static/src/stylesheets/module/content/_truncation.scss]: this is a very expensive thing to render, it looks like a bit one too. bit nervous about it...

Comment (OliverJAsh) [inline static/src/stylesheets/module/content/_truncation.scss]: As long as we don't need to re-{render,paint} all the time it should be OK?

Comment (stephanfowler) [inline static/src/stylesheets/module/content/_truncation.scss]: Interesting. Does a shadow cause repaints (eg on scroll) if the element is just in the normal page flow (ie not fixed pos)? BTW I changed it from a 200px gradient - dunno if that is worse or better.

Comment (stephanfowler) [inline static/src/stylesheets/module/content/_truncation.scss]: I've replaced this with a opacity gradient over 40px.

**Comment by janua** [comment]
:+1:

**Pull request merged**